examples/plugins/aggregate/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 3.18...3.22 FATAL_ERROR)
+
+project(
+  example-aggregate
+  DESCRIPTION "Example aggregate transform plugin for VAST"
+  VERSION 0.1
+  LANGUAGES CXX)
+
+# Enable unit testing. Note that it is necessary to include CTest in the
+# top-level CMakeLists.txt file for it to create a test target, so while
+# optional for plugins built alongside VAST, it is necessary to specify this
+# line manually so plugins can be linked against an installed VAST.
+include(CTest)
+
+find_package(VAST REQUIRED)
+VASTRegisterPlugin(
+  TARGET example-aggregate
```

Comment:
This needs to be manually enabled in CI (`.github/workflows/vast.yaml`) under `jobs.build-plugins.strategy.matrix.plugin`.

Reply:
Instead of an example aggregate plugin, we created an internal, more aggregate-like VAST plugin (`count`), making it easier to write unit and integration tests.